engine/src/main/java/org/terasology/rendering/dag/nodes/ToneMappingNode.java

```diff
@@ -65,8 +65,13 @@ public ToneMappingNode(String nodeUri, Context context) {
 
         screenGrabber = context.get(ScreenGrabber.class);
 
-        DisplayResolutionDependentFBOs displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFBOs.class);
-        FBO toneMappingFbo = requiresFBO(new FBOConfig(TONE_MAPPING_FBO_URI, FULL_SCALE, FBO.Type.HDR), displayResolutionDependentFBOs);
+        DisplayResolutionDependentFBOs displayResolutionDependentFboManager = context.get(DisplayResolutionDependentFBOs.class);
+        FBO toneMappingFbo = requiresFbo(new FBOConfig(TONE_MAPPING_FBO_URI, FULL_SCALE, FBO.Type.HDR), displayResolutionDependentFboManager);
+
+        addOutputFboConnection(1,toneMappingFbo);
```

Comment:
add output connection inside this node
setDependencies not yet used, should be inside TODO